.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build-test:
+    runs-on: ubuntu-18.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['7.8', 'current']
```

Comment:
We should either not test on 7.8 or allow it to fail (it wouldn't work currently).

Reply:
HEAD needs changes to gui and DrRacket and a new package that's not part of 7.9 even.